internal/builders/go/pkg/build.go

```diff
@@ -116,25 +120,41 @@ func (b *GoBuild) Run(dry bool) error {
 		// Generate the command.
 		com := b.generateCommand(flags, filename)
 
-		// Share the resolved name of the binary.
-		fmt.Printf("::set-output name=go-binary-name::%s\n", filename)
-		command, err := utils.MarshalToString(com)
+		env, err := b.generateCommandEnvVariables()
 		if err != nil {
 			return err
 		}
-		// Share the command used.
-		fmt.Printf("::set-output name=go-command::%s\n", command)
 
-		env, err := b.generateCommandEnvVariables()
+		r := runner.CommandRunner{
+			Steps: []*runner.CommandStep{
+				{
+					Command:    com,
+					Env:        env,
+					WorkingDir: dir,
+				},
+			},
+		}
+
+		steps, err := r.Dry()
 		if err != nil {
 			return err
 		}
 
-		menv, err := utils.MarshalToString(env)
+		menv, err := utils.MarshalToString(steps[0].Env)
```

Comment:
I know it' obvious, but maybe add a comment that there's a single command in the steps, which is why we're using `[0]`

Reply:
Done.